src/Components/build/arcade.props

```
@@ -0,0 +1,37 @@
+<!--
+These settings align older versions of KoreBuild with the layout expected by arcade tooling.
+This files is a temporary while this project transitions to using arcade.
+-->
+<Project>
+
+  <!--
+    Project output layout.
+
+    These properties overrdie the default locations to obj and bin directories.
+  -->
+
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ArtifactsDir Condition="'$(ArtifactsDir)' == ''">$(RepositoryRoot)artifacts\</ArtifactsDir>
+    <ArtifactsObjDir>$(ArtifactsDir)obj\</ArtifactsObjDir>
+    <ArtifactsConfigurationDir>$(ArtifactsDir)$(Configuration)\</ArtifactsConfigurationDir>
+    <ArtifactsBinDir>$(ArtifactsConfigurationDir)bin\</ArtifactsBinDir>
+    <PackageOutputPath>$(ArtifactsConfigurationDir)packages\</PackageOutputPath>
```

```suggestion
    <PackageOutputPath>$(ArtifactsDir)build\</PackageOutputPath>
```

Changing this line should fix the PR build failure.

---

Thanks - done